operator/src/monitoring/prom-config.yaml

```diff
@@ -0,0 +1,4 @@
+global:
+  scrape_interval: 10s
+  scrape_timeout: 5s
```

Comment:
These seem a bit aggressive to me. Do we need near real-time metrics for our use cases? Can we scrape every 30 seconds to a minute?

Reply:
I just pulled these setting along from the existing config.
Want to drop them down to the defaults?

```
# How frequently to scrape targets by default.
  [ scrape_interval: [<duration>](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#duration) | default = 1m ]

  # How long until a scrape request times out.
  [ scrape_timeout: [<duration>](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#duration) | default = 10s ]
  ```